.changeset/rare-frogs-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-svelte": patch
```

Comment:
changes in tests don't require changesets. please remove this file

Reply:
It changes a file that is shipped with create-svelte, so the version should be bumped in this case.

Reply:
oh, my bad. thanks for checking @gtm-nayan